utilities/transactions/write_unprepared_txn.h

```diff
@@ -199,6 +208,31 @@ class WriteUnpreparedTxn : public WritePreparedTxn {
   // but in some cases, we should be able to restore the previously largest
   // value when calling RollbackToSavepoint.
   SequenceNumber largest_validated_seq_;
+
+  struct SavePoint {
+    // Record of unprep_seqs_ at this savepoint. The set of unprep_seq is
+    // used during RollbackToSavepoint to determine visibility when restoring
+    // old values.
+    //
+    // Since all unprep_seqs_ sets further down the stack must be subsets, this
+    // can potentially deduplicated by just storing set difference.
```

Comment:
There isn't necessarily a TODO, since I don't know if spending extra time computing set difference is worth the memory savings (especially if you don't expect unprep_seqs to be that large).

If it's too confusing, I can just get rid of the comment.